flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/DisplayMaxColumnWidthSQLExample.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.examples.java.basics;
+
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.Table;
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.api.config.TableConfigOptions;
+import org.apache.flink.types.Row;
+
+import java.time.LocalDate;
+
+import static org.apache.flink.table.api.Expressions.range;
+import static org.apache.flink.table.api.Expressions.withColumns;
+
+/**
+ * Example for using {@link TableConfigOptions#DISPLAY_MAX_COLUMN_WIDTH}.
+ *
+ * <p>The example shows how to set {@link TableConfigOptions#DISPLAY_MAX_COLUMN_WIDTH} show more or
+ * less content of columns with variable-length types (e.g. STRING) in both batch and streaming
+ * mode.
+ *
+ * <p>The example executes two Flink jobs. The results are written to stdout.
+ */
+public final class DisplayMaxColumnWidthSQLExample {
```

Comment:
I'm pretty confused about adding this example. This example is very different from others. Examples are usually used to demonstrate a specific use case or show how to use a significant feature. I'm not sure we need to add examples for each configuration. If this is for verifying, UT and IT are enough.

Reply:
I did find the right place to do IT for TableConfigOption. Any suggestion?

Reply:
I think documentation is enough. Examples jars will be placed under `<FLINK_HOME>/examples`, so they are used as `flink run` to submit to Flink clusters instead of running in IDE.